source/usage-examples/distinct.txt

```diff
@@ -0,0 +1,48 @@
+==========================================
+Distinct Values of a Field in a Collection
+==========================================
+
+.. default-domain:: mongodb
+
+You can retrieve a list of distinct values for a field across a collection
+by using the
+`distinct()
+<https://mongodb.github.io/node-mongodb-native/3.4/api/Collection.html#distinct>`_
+method. The ``distinct()`` method is called on a collection object and
```

Comment:
Passive sentence, you could change this to:
```
Call the ``distinct()`` method on a Collection object with a document field name to produce a list...
```
which also flows nicely into the next sentence, which currently starts by repeating the gist of this sentence.

Reply:
Thanks, will fix.